app/controllers/hq/partners_controller.rb

```diff
@@ -1,71 +1,44 @@
 class Hq::PartnersController < ApplicationController
-  before_action :authenticate_admin_user!
+  before_action :authenticate_admin_user!,
+                :new_partner_facade
   layout 'application'
 
   def index
-    @partners= Partner.all
+    @partners = Partner.all
   end
```

Comment:
I think `@partners_facade.partners` is a good idea.  We mustn't assume we will be sending all the partners to the view.  In the three sponsors index views, there are different subsets of sponsors.  What about `load_partners :all` where `:all` is a message to the model.  So you could call `load_partners :active` if you wanted to.